.github/workflows/build_steps.yml

```diff
@@ -177,7 +177,7 @@ jobs:
       # ========================= Common =========================
       - name: Disk usage
         if: always()
-        run: du -m . ${{matrix.build_dir}} ${{matrix.vcpkg_packages_dir}} | sort -n | tail -n 50; df -h
+        run: du -m . ${{matrix.build_dir}} ${{matrix.vcpkg_packages_dir}} | sort -n | tail -n 50 || true; df -h
```

Comment:
What could fail here?

Reply:
If those directories haven't been created yet then the `du` will fail and we don't get to inspect the `df` output.